src/Native/System.Net.Security.Native/pal_ntlmapi.cpp

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "pal_types.h"
+#include "pal_ntlmapi.h"
+#include "pal_utilities.h"
+#include <stddef.h>
+#include <assert.h>
+#include <time.h>
+#include "heimntlm.h"
+#include "openssl/hmac.h"
```

Comment:
Presumably the more stylistically-correct way would be
```cpp
#include <openssl/hmac.h>
```

Since it's coming from the system includes.